kiwi/defaults.py

```diff
@@ -443,19 +443,20 @@ def get_runtime_checker_metadata() -> Dict:
             return yaml.safe_load(meta)
 
     @staticmethod
-    def get_exclude_list_from_custom_exclude_files(root_dir: str) -> List:
+    def _parse_exclude_file(root_dir: str, exclude_filename: str) -> List:
         """
         Provides the list of folders that are excluded by the
-        optional metadata file image/exclude_files.yaml
+        optional metadata file
```

Comment:
This method's description should focus on the exclude list format. Also, the file is not optional within the context of this method.

```suggestion
        Retrieves an exclusion list from the provided metadata file

        The file should contain a YAML dictionary with a top-level key
        named 'exclude' and the list of exclusions as its value.

        The list of exclusions may include:
            * file paths
            * folder paths
            * glob patterns

        Paths and patterns should be relative to the root of the
        filesystem that they're being excluded from.
```